src/lib/db.js

```diff
@@ -1578,3 +1442,110 @@ export function getSemesterTeams(semesterName, teamIndices) {
     })
   ));
 }
+
+export async function getNumArticles() {
+  const rows = await database('articles').count('* as numArticles');
+  return rows[0].numArticles;
+}
+
+/**
+ * Updates which category an article belongs to
+ * @param {Object} jsonGraphArg - An object of type { [slugs]: articleUpdateObject[] }
+ * @returns {Promise<string[]>} - Slugs of the articles that had their category changed
+ */
+async function updateArticleCategories(jsonGraphArg) {
+  // This variable filters out all articles that don't have a category update requested
```

Comment:
@frederikbrinck is this clearer now? Though as I write this I realize also the variable probably doesn't filter itself, but it stores the filtered list, so maybe some grammar I can quickly fix, but other than that?